package.json

```diff
@@ -29,7 +29,9 @@
       "hydrogen:run-cell-and-move-down"
     ]
   },
-  "scripts": {},
+  "scripts": {
+    "install": "npm rebuild --build-from-source"
```

Comment:
Good point I think I only removed `node_modules/zmq-prebuilt/` during testing. I test this.